etl/steps/data/meadow/happiness/2024-06-09/happiness.py

```diff
@@ -0,0 +1,37 @@
+"""Load a snapshot and create a meadow dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshot.
+    snap = paths.load_snapshot("happiness.xls")
+
+    # Load data from snapshot.
+    tb = snap.read()
+
+    # add year and country columns
+    tb["year"] = 2023  # 2024 report -> 2023 data
+    tb["country"] = tb["Country name"]
+    tb = tb.drop(columns=["Country name"])
```

Comment:
Instead you could simply do `tb = tb.rename(columns={"Country name": "country"}, errors="raise")`.